extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -479,4 +476,19 @@ export class AmpSlideScroll extends BaseSlides {
       this.slidesContainer_./*OK*/scrollLeft = interpolate(pos);
     }, 80, 'ease-out').thenAlways();
   }
+
+  /**
+   * Cancels the touchmove events for the element so that viewer does not
+   * consider the swipes in the carousel as swipes for changing AMP documents.
+   * @private
+   */
+  cancelTouchEvents_() {
+    // TODO(aghassemi): https://github.com/ampproject/amphtml/issues/4742
+    // prevents us from using SwipeXRecognizer with an empty handler to
+    // cancel the events. This work around is not great and temporary until
+    // #4742 is fixed.
+    this.element.addEventListener('touchmove', evt => {
+      evt.stopPropagation();
+    });
```

Comment:
also consider if we should just do it here 
https://github.com/aghassemi/amphtml/blob/23c3bb8d891e47fc522d45468401ef443add5c6e/extensions/amp-carousel/0.1/slidescroll.js#L120